components/profile/ProfileDetail.vue

```diff
@@ -187,7 +187,9 @@
         <div
           v-if="userProfile?.description"
           class="max-w-lg whitespace-break-spaces text-sm">
-          {{ userProfile.description }}
+          <Markdown
+            :source="userProfile.description"
+            data-testid="profile-description" />
```

Comment:
I would remove the testid as it's not used.
```suggestion
            :source="userProfile.description" />
```

Reply:
can be tested by @prury for e2e tests

Reply:
Yes, but there is no e2e for this component/page for now, so the `testid` is not used. Once we introduce the e2e test case for it, we could add it. To avoid confusion.